modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-grid/src/main/resources/META-INF/resources/grid_field.js

```diff
@@ -71,53 +55,60 @@ AUI.add(
 					getValue: function() {
 						var instance = this;
 
-						var gridRowsNode = instance.getGridRowsNode();
```

Comment:
Instead of access the DOM to get the selected value you can use the `value` attribute, since we guarantee the `value` is always updated.